client/extensions/woocommerce/app/settings/payments/payment-method-edit-form-toggle.js

```diff
@@ -21,6 +23,7 @@ const PaymentMethodEditFormToggle = ( { checked, name, onChange } ) => {
 
 	return (
 		<FormToggle
+			{ ...omit( props, [ 'updateAriaLabel', 'editAriaLabel' ] ) }
```

Comment:
Oops - the array here is any props you don't want passed along, which is probably just `onChange` since you set a handler below.